test/unit/processing/calculation/get-calculation.test.js

```diff
@@ -24,26 +24,28 @@ const getCalculationByPaymentRequestId = require('../../../../app/processing/cal
 
 const getCalculation = require('../../../../app/processing/calculation/get-calculation')
 
-let calculation
-let rawCalculationData
-const paymentRequest = require('../../../mock-payment-request').processingPaymentRequest
-const paymentRequestId = paymentRequest.paymentRequestId
-const invoiceNumber = paymentRequest.invoiceNumber
+let paymentRequestId
+let invoiceNumber
+let retreivedCalculation
 
 describe('get and transform payment request information for building a statement object', () => {
   beforeEach(() => {
-    const retrievedCalculationData = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-calculation').rawCalculationData))
-
-    rawCalculationData = retrievedCalculationData
-    calculation = {
-      sbi: rawCalculationData.sbi,
-      calculated: new Date(rawCalculationData.calculationDate),
-      invoiceNumber: rawCalculationData.invoiceNumber,
-      paymentRequestId: rawCalculationData.paymentRequestId
+    const calculation = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-calculation')))
+    const paymentRequest = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-payment-request').processingPaymentRequest))
+
+    paymentRequestId = paymentRequest.paymentRequestId
+    invoiceNumber = paymentRequest.invoiceNumber
+
+    retreivedCalculation = {

```

Comment:
Should `retreivedCalculation` be set up as a constant as well?
Also noticed retreived is spelt incorrectly, should be retrieved